code/WorkInProgress/zamu_mail.dm

```diff
@@ -628,10 +756,52 @@ var/global/mail_types_by_job = list(
 		)
 	)
 
+// =========================================================================
+// Various items given out based on player trait choices, to make it more personal
+var/global/list/mail_types_by_trait = list(
```

Comment:
Global lists like this are bad for maintainability, these should be on the traits' `typeinfo`s (to avoid re-initializing for every trait instance)